tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -256,7 +256,7 @@ def test_read_csv(session, mode):
     assert res == [Row(1, "one", 1.2), Row(2, "two", 2.2)]
 
     with pytest.raises(SnowparkDataframeReaderException):
-        session.read.csv(test_file_on_stage)
```

Comment:
Better add a new test for this one since it doesn't raise an exception any more.